tap_logmeinrescue/streams/technicians.py

```diff
@@ -60,3 +64,4 @@ def sync_data(self, return_ids=False):
             LOGGER.info("Syncing {}".format(substream.TABLE))
             substream.sync_data(
                 parent_ids=technician_ids)
+            self.state = substream.state
```

Comment:
We were dropping state as we went through the run by not copying the substream's state back into the technicians stream.